docs/source/data_user_agreement.rst

```diff
@@ -3,19 +3,23 @@
 Data User Agreement Template
 ============================
 
-This template is based on the `Donder’s Institute DUA Version RU-DI-HD-1.0. <https://data.donders.ru.nl/doc/dua/RU-DI-HD-1.0.html?2>`_. Given that brain imaging data can be used to single out individuals, it is safe to consider them as 'personal data' under `GDPR <https://gdpr-info.eu/>`_, even after using a pseudo-anonymization procedure like defacing. To allow easy sharing, it is recommended to use a Data User Agreement (DUA) over a license.
-
+.. _Data User Agreement (DUA):
 
+This template is based on the `Donder’s Institute DUA Version RU-DI-HD-1.0. <https://data.donders.ru.nl/doc/dua/RU-DI-HD-1.0.html?2>`_.
+Given that brain imaging data can be used to identify individuals, it is safe to consider them as 'personal data' under the `GDPR <https://gdpr-info.eu/>`_, even after using a pseudo-anonymization procedure like defacing.
```

Comment:
I like that idea. And then we can also point to the links on this site where tools and processes are provided.

Reply:
I don't think we should bring much attention to that - since the data are personnal and under DUA, there is no need to actually deface anymore (and many have shown this can impact segmentation and of course useless for MEG-EEG source reconstruction)

Reply:
I don't fully agree. A data protection office could still argue that we (data collectors/controllers) should minimise the risk of reidentification even if a DUA exists. It's all about the balance of risks, since even with a DUA, people could still act outside of its bounds and then the risk of reidentification is higher without defacing. Of course, for cases such as MEG-EEG source reconstruction this risk balance could shift in a different direction.

Regarding extra info about defacing etc, I think that would be useful but perhaps more applicable on the page where anonymization techniques are given currently. On this specific line in the DUA we could perhaps just put a link to the anonymization page, for users who want to know more.

Reply:
ok I'll add ... even after using a pseudo-anonymization procedure like defacing (see our information on `(peudo)anonymization tools <https://github.com/CPernet/open-brain-consent/blob/GLiMR-workshop/docs/source/anon_tools.rst>`_). ??

Reply:
Sounds about right! A quick (very brash) self-advertisement: [BIDSonym](https://github.com/PeerHerholz/BIDSonym) now also supports [mridefacer](https://github.com/mih/mridefacer) and has a [documentation](https://peerherholz.github.io/BIDSonym/).